app/decorators/casa_case_decorator.rb

```diff
@@ -25,6 +25,20 @@ def case_contacts_ordered_by_occurred_at
     object.case_contacts.sort_by(&:occurred_at)
   end
 
+  def case_contacts_latest
+    object.case_contacts.max_by(&:occurred_at)
```

Comment:
This should be a scope on the model, to avoid having to pull all the case contacts every time.